### DISCUSSION TITLE
Support alternative container tools - podman, nerdctl

### DISCUSSION THREAD

**Discussion by planetf1** [JetBrains/Qodana#144] [Ideas]
I have docker aliases to 'nerdctl' - along with rancher desktop

However qodana does not work

```
  main git:(main) alias docker
docker=nerdctl
➜  main git:(main) docker ps
CONTAINER ID    IMAGE    COMMAND    CREATED    STATUS    PORTS    NAMES
➜  main git:(main) qodana scan --show-report
✗  'docker ps' exited with exit code 1, perhaps docker daemon is not running?
➜  main git:(main)
```

qudana should allow these alternative container tools to be used

**Comment by brichbash** [comment]
Hello,

Thank you for the request!

We have an issue on our tracker about nerdctl support and also I've created one about Podman. If you have a spare minute, please vote for them ([QD-3341](https://youtrack.jetbrains.com/issue/QD-3341/Support-nerdctl), [QD-4729](https://youtrack.jetbrains.com/issue/QD-4729/Support-Podman)) to get notifications when there are any updates.
